src/cbc_sdk/platform/alerts.py

```diff
@@ -1618,3 +1634,30 @@ def _perform_query(self, from_row=1, max_rows=-1):
             if current >= self._total_results:
                 still_querying = False
                 break
+
+    def close(self, closure_reason=None, determination=None, note=None, ):
```

Comment:
I'm confused why we want to implement this in GroupedAlertSearchQuery as it only covers a single workflow update and affects all alerts not just the alert of a single grouped alert. It seems like we'd rather have users call the code themselves
```
alert_query = self.get_alert_search_query()
alert_query.close(closure_reason, note, determination)
```

Reply:
That's fair. Have changed it to raise a NotImplemented exception.
It needs something to override the Alert implementation.